cub/cmake/CubCudaConfig.cmake

```diff
@@ -23,7 +23,14 @@ list(FILTER CUB_CUDA_ARCHITECTURES_RDC EXCLUDE REGEX "53|62|72|90")
 message(STATUS "CUB_CUDA_ARCHITECTURES:     ${CUB_CUDA_ARCHITECTURES}")
 message(STATUS "CUB_CUDA_ARCHITECTURES_RDC: ${CUB_CUDA_ARCHITECTURES_RDC}")
 
-option(CUB_ENABLE_RDC_TESTS "Enable tests that require separable compilation." ON)
+if ("MSVC" STREQUAL "${CMAKE_CXX_COMPILER_ID}")
```

Comment:
```suggestion
if (MSVC)
```

Reply:
I didn't know there was a short version for this check, thank you! Since it's functionally equivalent and there are many usages of the long version, I'll keep the code as is, but will keep this in mind.